<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnHit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Hit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStand">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stand</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSurrender">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Surrender</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblInfo">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>40</y>
      <width>281</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblPlayer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Player</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDealer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dealer</string>
    </property>
   </widget>
   <widget class="QWidget" name="widgetP" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>531</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widgetD" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>531</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblAmount">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>220</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblNum">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>220</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>480</y>
      <width>71</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Your balance: &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Amount bet:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>480</y>
      <width>18</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="lblBalance">
       <property name="text">
        <string>0.0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblCurrentBet">
       <property name="text">
        <string>0.0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>543</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRules">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionRules"/>
   </widget>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionNew_Game"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuRules"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRules">
   <property name="text">
    <string>Rules</string>
   </property>
  </action>
  <action name="actionNew_Game">
   <property name="text">
    <string>New Game</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
